py/desispec/zcatalog.py

```diff
@@ -157,6 +157,13 @@ def _get_survey_program_from_filename(filename):
     program = arr[2]
     return survey, program
 
+def create_summary_catalog_wrapper(kwargs):
+    """
+    Trivial wrapper around create_summary_catalog that takes a dict
+    and passes the key-value pairs to create_summary_catalog as keyword arguments
+    """
+    return create_summary_catalog(**kwargs)
```

Comment:
It appears that this function exists only so that `desispec.scripts.zcatalog_wrap` can call it with `runcmd(create_summary_catalog_wrapper, args=kwargs, ...)` because `runcmd` only supports a list of `args` and not a dict of `kwargs`.  Ideally `runcmd` would also support `kwargs`, but that is admittedly kind of messy given everything else that it is doing (function vs. script, MPI vs. not...).  Lacking that, I suggest moving this function into `desispec/scripts/zcatalog_wrap.py` as a `_hidden` function for this one use-case and not making it part of the `desispec.zcatalog` namespace.

Reply:
I couldn't agree more about your `runcmd` comments. That was what I initially wanted to do before realizing it didn't accept kwargs. That change would be subtle and feature creep on this PR, so I've taken the second suggestion of making it a _hidden function in `zcatalog_wrap.py`.